cli/cmd/plugin/unmanaged-cluster/packages/packages.go

```diff
@@ -91,6 +91,10 @@ type PackageManager interface {
 	// referencing the cluster-admin CluterRole. This essentially provides full admin access to anything
 	// referencing this service account. Upon success, it returns the created ServiceAccount.
 	CreateRootServiceAccount(ns, name string) (*v1.ServiceAccount, error)
+	// GetRootServiceAccount returns the root service account
+	// given a namespace and the name of the service account
+	// Fails if service account doesn't exist
+	GetRootServiceAccount(ns, name string) (*v1.ServiceAccount, error)
```

Comment:
nit: Probably best to return `nil` if a service account isn't found.

reasoning: if the service count isn't there it isn't an error of GetRootServiceAccount (like a failure to communicate with the cluster would be), it's an ok behavior and the caller should figure out if it wants to bubble up an error based on `nil` being returned.